Dockerfile

```diff
@@ -10,13 +10,10 @@ COPY consoles/                              /usr/share/prometheus/consoles/
 RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
 RUN mkdir -p /prometheus && \
     chown -R nobody:nogroup etc/prometheus /prometheus
+RUN ln -s /prometheus /etc/prometheus/data
```

Comment:
Every docker `RUN` command creates a new layer; you should combine this in to a single command.

Reply:
O.o WTF, Thanks docker.